mongonaut/views.py

```diff
@@ -96,8 +96,10 @@ def get_queryset(self):
             self.page = 1
 
         obj_count = queryset.count()
-        self.total_pages = obj_count / self.documents_per_page + (1 if obj_count % self.documents_per_page else 0)
-
+        self.total_pages = int(
```

Comment:
I'm pretty sure this is caused by Python's implementation changing for type casting during arithmetic. 

I'd prefer if this did a ceiling call rather than cast to int. https://docs.python.org/3/library/math.html#math.ceil

So this would likely change to: 

```python
import math

#####

self.total_pages = math.ceil(obj_count / self.documents_per_page)
```

My quick check in the console seems to agree on Python 3.5.1

```python
>>> math.ceil(99/25)
4
>>> math.ceil(101/25)
5
>>> math.ceil(100/25)
4
```